contracts/core/lib/auction-price-libraries/LinearAuctionPriceCurve.sol

```diff
@@ -134,10 +144,13 @@ contract LinearAuctionPriceCurve {
 
         // If time hasn't passed to pivot use the user-defined curve
         if (elapsed <= _auctionParameters.auctionTimeToPivot) {
-            // Calculate the priceNumerator as a linear function of time between 0 and _auctionPivotPrice
-            priceNumerator = elapsed
-                .mul(_auctionParameters.auctionPivotPrice)
-                .div(_auctionParameters.auctionTimeToPivot);
+            // Calculate the priceNumerator as a linear function of time between _auctionStartPrice and
+            // _auctionPivotPrice
+            priceNumerator = auctionStartPrice.add(
```

Comment:
Is it possible to break up the calculations (particularly the nested ones) for readability sake?